### DISCUSSION TITLE
Transfer to PDF in page printing mode

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#858] [Q&A]
in ngrinder 3.5.5,After opening the detailed test results, you want to print the page, but there is a problem with the style and the display is incomplete,why?

![image](https://user-images.githubusercontent.com/50607317/146132704-72c04844-593b-46ad-99a4-fe5d18524588.png)
@imbyungjun @junoyoon 
thks

**Comment by imbyungjun** [comment]
Hello, @zhougy1989
I'm busy with another project right now, and it looks not that easy to fix.
How about using a Chorme plugin like [gofullpage](https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl) to capture the whole page?
